Documentation/devel/stage1-implementors-guide.md

```diff
@@ -50,10 +50,10 @@ An alternative stage 1 could forego systemd-nspawn and systemd altogether, or re
 All that is required is an executable at the place indicated by the `coreos.com/rkt/stage1/run` entrypoint that knows how to apply the pod manifest and prepared ACI file-systems to good effect.
 
 The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`.
-Stage 1 must write the host PIDs of the pod's process #1 and that process's parent to these two files, respectively:
+Stage 1 implementors have two options for doing so; only one must be implemented:
```

Comment:
Only? Or maybe "at least". But i'm not sure if it's true - in kvm flavor i was writing into `ppid` file, and this led to hang in endless loop, in entering process.